packages/tools/utils-react-widgets/package.json

```diff
@@ -30,7 +30,7 @@
     "cross-env": "^7.0.2",
     "css-loader": "^5.0.0",
     "del": "^6.0.0",
-    "eslint": "^7.12.1",
+    "eslint": "^7.20.0",
```

Comment:
Why do we need to explicitly bump this? I feel like if rollup replace introduces ESLint 7.20, it would be picked up by the other packages as well right since they match the semver

Reply:
Just to guarantee, as `@rollup/plugin-replace` introduces this in their deps